jmclient/jmclient/taker.py

```diff
@@ -485,12 +485,11 @@ def receive_utxos(self, ioauth_data):
         #we have tried to avoid this based on over-estimating the needed amount
         #in SendPayment.create_tx(), but it is still a possibility if one maker
         #uses a *lot* of inputs.
-        if self.my_change_addr and my_change_value <= 0:
+        if my_change_value < 0:
             raise ValueError("Calculated transaction fee of: " +
                 btc.amount_to_str(self.total_txfee) +
                 " is too large for our inputs; Please try again.")
-        elif self.my_change_addr and my_change_value <= jm_single(
-        ).BITCOIN_DUST_THRESHOLD:
+        if self.my_change_addr and my_change_value <= jm_single().BITCOIN_DUST_THRESHOLD:
```

Comment:
No need for this here (discussions about PEP8 - let's just leave them orthogonal, maybe open an issue about the whole formatting question, something like that).

Reply:
Agreed WRT the style. I will make a separate issue for that. But the `elif` -> `if` change is needed to enable dropping the change output when change is expected but `my_change_value` is 0.